app/db.py

```
@@ -0,0 +1,80 @@
+    ):
+        url = getattr(settings, attr, None)
+        if url:
+            candidates.append((url, label))
+
+    last_err: Exception | None = None
+    for url, label in candidates:
+        try:
+            log.info("Trying %s database connection ...", label)
+            pool = await asyncpg.create_pool(url)
+            log.info("Connected via %s", label)
+            return pool
```

Ruff (0.15.4)

[warning] Consider moving this statement to an `else` block

(TRY300)